src/app/loginComponents/requests/organization-invite-confirm-dialog.html

```diff
@@ -10,9 +10,23 @@ <h1 mat-dialog-title>{{data.approve ? 'Accept' : 'Decline'}} Invite to {{data.na
   <div fxFlex></div>
   <button mat-button (click)="onNoClick()">No Thanks</button>
   <span *ngIf="data.approve; else rejectBlock">
-    <button mat-flat-button id="accept-pending-org-dialog" [mat-dialog-close]="{ 'id': data.id, 'approve': data.approve }">Accept</button>
+    <button
+      mat-flat-button
+      id="accept-pending-org-dialog"
+      data-cy="approve-pending-org-dialog"
```

Comment:
These ids were added by someone else. I did not add these ids so they were used in other code unrelated to this one. I only added the `data-cy` attributes in these files to avoid using its already existing `id`